internal/slicer/slicer_test.go

```diff
@@ -1088,24 +1088,7 @@ func (a *testArchive) Exists(pkg string) bool {
 }
 
 func (s *S) TestRun(c *C) {
-	// Run tests for format chisel-v1.
 	runSlicerTests(c, slicerTests)
-
-	// Run tests for format v1.
-	v1SlicerTests := make([]slicerTest, len(slicerTests))
-	for i, t := range slicerTests {
-		t.error = strings.Replace(t.error, "chisel-v1", "v1", -1)
-		t.error = strings.Replace(t.error, "v1-public-keys", "public-keys", -1)
-		m := map[string]string{}
-		for k, v := range t.release {
-			v = strings.Replace(v, "chisel-v1", "v1", -1)
-			v = strings.Replace(v, "v1-public-keys", "public-keys", -1)
-			m[k] = v
-		}
-		t.release = m
-		v1SlicerTests[i] = t
-	}
-	runSlicerTests(c, v1SlicerTests)
 }
 
 func runSlicerTests(c *C, tests []slicerTest) {
```

Comment:
Can you inline this, I think we talked about inlining the functions in the tests now and I see we did it for `setup_test.go` already.

Reply:
Done: 2f22f6e.